sdk/keyvault/keyvault-keys/src/generated/keyVaultClient.ts

```diff
@@ -703,6 +715,48 @@ class KeyVaultClient extends KeyVaultClientContext {
       callback) as Promise<Models.UnwrapKeyResponse>;
   }
 
+  /**
+   * The export key operation is applicable to all key types. The target key must be marked
+   * exportable. This operation requires the keys/export permission.
+   * @summary Exports a key.
+   * @param vaultBaseUrl The vault name, for example https://myvault.vault.azure.net.
+   * @param keyName The name of the key to get.
+   * @param keyVersion Adding the version parameter retrieves a specific version of a key.
+   * @param environment The target environment assertion.
+   * @param [options] The optional parameters
+   * @returns Promise<Models.ExportKeyResponse>
+   */
+  exportKey(vaultBaseUrl: string, keyName: string, keyVersion: string, environment: string, options?: coreHttp.RequestOptionsBase): Promise<Models.ExportKeyResponse>;
```

Comment:
You just changed all the options classes but defined this using the original options class. Maybe derive it now to avoid issues down the road?